packages/core/contracts/financial-templates/common/financial-product-libraries/CoveredCallFinancialProductLibrary.sol

```diff
@@ -28,7 +27,6 @@ contract CoveredCallFinancialProductLibrary is FinancialProductLibrary, Ownable,
      */
     function setFinancialProductStrike(address financialProduct, FixedPoint.Unsigned memory strikePrice)
         public
```

Comment:
Can you change the `Note` which now improperly references how only the owner can set the strike? Also, if you can add a note like you did in the description about the recommended usage, that'd be great: i.e. caller should set strike after deployment but before depositing the synthetic token into a liquidity pool